site/content/docs/4.3/getting-started/javascript.md

```diff
@@ -64,6 +64,19 @@ myModal.addEventListener('show.bs.modal', function (e) {
 })
 {{< /highlight >}}
 
+{{< callout warning >}}
+## jQuery events
+
+Bootstrap detect jQuery only if `jQuery` is present in `window` object. If jQuery is found, Bootstrap will emit events thanks to jQuery event system.
```

Comment:
```suggestion
Bootstrap will detect jQuery only if `jQuery` is present in `window` object. When jQuery is detected, Bootstrap will emit events thanks to jQuery event system.
```

Reply:
Thanks. I noticed a few other things in this sentence.